### PR TITLE
[B] Use span tags for list item titles

### LINKED ISSUE #2002: Backend list item text should not be wrapped with h3s
List of subjects in subjects page is a list of headings, but these elements should not be headings as they do not contribute semantic information

### PR DESCRIPTION
**Pull request by SMaxOwok**
Fixes #2002

### CODE REVIEW

File: client/src/backend/components/subject/ListItem.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default class SubjectListItem extends PureComponent {
         <Link to={lh.link("backendSettingsSubject", subject.id)}>
           <header>
```
Comment (zdavis) [inline client/src/backend/components/subject/ListItem.js]: Seems like this <header> tag is also semantically incorrect and should be removed.

File: client/src/backend/components/user/__tests__/__snapshots__/ListItem-test.js.snap [hunk 2]
```diff
@@ -29,13 +29,13 @@ exports[`Backend.User.ListItem component renders correctly 1`] = `
       <div
         className="meta"
       >
-        <h3
+        <span
```
Comment (zdavis) [inline client/src/backend/components/user/__tests__/__snapshots__/ListItem-test.js.snap]: This is wrapped in a <header> tag as well, higher up.

File: client/src/backend/components/user/ListItem.js [hunk 3]
```diff
@@ -56,9 +56,9 @@ export default class UserListItem extends PureComponent {
               )}
             </figure>
             <div className="meta">
-              <h3 className="name large">
+              <span className="name large">
```
Comment (zdavis) [inline client/src/backend/components/user/ListItem.js]: Header tag on line 40 should be removed as well?

**Comment by SMaxOwok** [comment]
Looks like all of our vertical list item components are using these tags/styles.  <= Note to self

**Pull request closed without merging**